packages/core/lib/commands/test/determineTestFilesToRun.js

```diff
@@ -1,27 +1,65 @@
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+
 const determineTestFilesToRun = ({ inputFile, inputArgs = [], config }) => {
-  const path = require("path");
-  const fs = require("fs");
-  const glob = require("glob");
   let filesToRun = [];
-
   if (inputFile) {
     filesToRun.push(inputFile);
-  } else if (inputArgs.length > 0) {
-    inputArgs.forEach(inputArg => filesToRun.push(inputArg));
+  }else if (inputArgs.length > 0) {
```

Comment:
Some Truffle style things; could you add spaces before and after your `else if`s like `} else if {` and not like `}else if{`?

Could you also add spaces in your `for` loop as well like `for (...) {` and not `for(...){`?

Additionally could you put your `else if`s on the same line as the closing bracket before it? So `} else if {` and not
```
}
else if {
```?